uploaddocs.sh

```diff
@@ -47,10 +47,10 @@ do
 
     # TODO: Product page
     echo "Generating metadata (googleapis.dev)"
-    python -m docuploader create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
+    dotnet run --project /tools/Google.Cloud.Tools.DocUploader -- create-metadata --name $pkg --version $version --language dotnet --github-repository googleapis/google-cloud-dotnet
```

Comment:
I think it would be better to give a repo-relative path here. Using /tools isn't ideal, IMO.

You could use `REPOROOT=$(git rev-parse --show-toplevel)` to obtain the repository root directory at the start of this script. (We might want to figure out if we could install this as a dotnet tool, but that's a different matter.)